pajbot/tmi.py

```diff
@@ -1,8 +1,33 @@
+from enum import Enum
+
+
+class WhisperOutputMode(Enum):
+    DISABLED = 0
+    NORMAL = 1
+    CHAT = 2
+
+    @staticmethod
+    def from_config_value(config_value):
+        return {
+            "disabled": WhisperOutputMode.DISABLED,
+            "normal": WhisperOutputMode.NORMAL,
+            "chat": WhisperOutputMode.CHAT,
+        }.get(config_value.lower(), WhisperOutputMode.NORMAL)
```

Comment:
```suggestion
        }.get(config_value.lower())
```

I'm suggesting it should just crash, that's easy to implement.